src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheckTest.java

```diff
@@ -108,25 +108,51 @@ public void testDefaultSettings()
     }
 
     @Test
-    public void testFirstSentence() throws Exception {
```

Comment:
Why not make the naming convention the same for test method and file/class name?

Reply:
You mean `testFirstSentance2a()` ?

Reply:
That seems to make more sense to me, or it would be even better to name the test by some defining characteristic of check config, i.e. `testFirstSentenceCheckHtmlFalseOne` or something.

Reply:
The `InputJavadocStyle` has total 10 input files which must be created into 40 different files. 

I am thinking to name inputs like:
2a, 2b, 2c, 2d
3a, 3b, 3c, 3d
4a, 4b, 4c, 4d..

And for test cases, just like you shared.

Reply:
To me, if I was a new contributor, I would wonder why we have test method like `testFirstSentenceOne` with test input `InputJavadocStyle2a.java`. These two names do not seem to be related. What I am getting at, is why not name the test method and input file similarly?

Example 1:

`testFirstSentenceOne` with input `InputJavadocStyleFirstSentenceOne.java`

Reply:
Or, even better, `testJavadocStyleFirstSentenceOne` and `InputJavadocStyleFirstSentenceOne.java`

Reply:
Yes, let's match test and input name